### DISCUSSION TITLE
Custom image resolution in DPI

### DISCUSSION THREAD

**Discussion by ricardomelo** [metafloor/bwip-js#344] [Q&A] [answered]
Hi!

Sorry if this is a dumb question, is it possible to set different DPI resolutions?

The documentation mentions that all images are rendered in 72 DPI but when printing, other DPI like 300 or 600 suits better. My goal is to generate QR Codes encoding GS1 Digital Link that will be sent to printers or printing services and according to GS1 Standards, those QR Code must have a X dimension of 0.495mm.

Playing with scale parameter, a factor of 5 gave me a module size quite close to the target, the question is if there exist an easy way to achieve this where we enable the user to choose both the DPI and Scale for perfect printing conditions.

Thanks for all the support!

**Comment by metafloor** [comment, marked as answer]
At present, there is no option to specify dpi.  Scale has always been the means of achieving higher resolution.  If you need a variable-resolution printing solution, you should give the svg rendering a test.  Most browsers scale them automatically to the printer's resolution.
